scripts/content.js

```diff
@@ -119,31 +124,24 @@ async function checkForCringe(post) {
     `;
 
     const prompt = `${SYSTEM_PROMPT_PREFIX} ${POST_CRITERIA}
-        If any of the above criteria are met, the tweet should be considered as a cringe post. Analyze this post and respond with ONLY "true" if the post is cringe-worthy or "false" if it's not. No other explanation needed. Max 1 word.
-
-        Respond EXCLUSIVELY using one of these formats:
+        If any criteria are met, respond with:
```

Comment:
lets not modify the suffix. model seems to be giving very descriptive respond using this prompt. we can take this up separately.